DESCRIPTION

```diff
@@ -33,10 +33,14 @@ Imports:
     glue,
     magrittr,
     purrr,
+    r2dii.plot,
     readr,
     rlang,
-    testthat,
+    stringr,
     tidyr,
     withr,
     zoo
+Remotes: 2DegreesInvesting/r2dii.plot
```

Comment:
Cool, just be aware that we are changing the API radically until the first release on CRAN. You may need some way to know if the interface changes in a way that breaks your code (i.e. an integration test -- like an .Rmd file that calls the r2dii.plot functions you want to use and breaks on CI if it fails to render.).

Reply:
I think it's fine for no, it's only being used in non-essential QA graphs for now. If they break, this won't impact any of the analysis. But I will keep my eyes open.